### DISCUSSION TITLE
Connections enable on/off

### DISCUSSION THREAD

**Discussion by Fumale** [frangoteam/FUXA#408] [Q&A]
Good morning,
My system consists of four machines that can be switched on or off randomly.
Is there a method to enable and disable the four opcua connections that communicate with the four machines automatically or manually from the graphical interface, not from the editor?
Regards
Alessandro

**Comment by superkalt** [comment]
why switching? if the system is offline, you will not get any data. Error "offline" will come.

![image](https://user-images.githubusercontent.com/44277174/173383659-4b880827-5224-4fd8-81ec-4157cec493d6.png)

**Reply by Fumale** [reply]
Can you indicate how to get the indication on the HMI of the status of the connections you posted?
Regards.

**Comment by superkalt** [comment]
it depends on the possible data. but you have to build a script that creates a tag or topic that is boolean type. if you get data it goes to "true" if not to "false"
